server/src/packages/pas-form/applicants/applicants.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Controller, Patch, Body, Param, Post } from '@nestjs/common';
+import { PasFormService } from '../pas-form.service';
+
+@Controller('applicants')
+export class ApplicantsController {
+  constructor(private readonly pasFormService: PasFormService) { }
+
+  @Patch('/:id')
+  patchApplicant(@Body() body, @Param('id') id) {
+    return {
+      dcp_applicantinformationid: id,
```

Comment:
Similar comment to above there will be also `dcp_applicantrepinformation` that has to be patched as well.